frontend/app/controllers/Joiner.scala

```diff
@@ -204,6 +204,8 @@ trait Joiner extends Controller with ActivityTracking with LazyLogging {
       case _ => Future.successful(None)
     }
 
+    val cookiesToDiscard = List(DiscardingCookie("GU_MEM"), DiscardingCookie("gu_paying_member", "/", Some(Config.guardianShortDomain)))
```

Comment:
As you're repeating the list in a few places anyway, I would probably just pass the arguments straight to `discardingCookies` rather than wrapping in a List and then unwrapping